matflow_damask/main.py

```diff
@@ -341,6 +356,60 @@ def modify_volume_element_add_buffer_zones(volume_element, buffer_sizes,
     return out
 
 
+@func_mapper(task='modify_volume_element', method='spread_orientations')
+def modify_volume_element_spread_orientations(volume_element, phase, max_spread_angle):
+    """
+    Include additional orientations within a limit around the orientation of a given phase.
+
+    append new idx and phase label to idx and phase label arrays, calculate oris within max range from given ori, append them to ori array. 
+    """
+    n_samples = 100
+
+    const_mat_idx = volume_element['constituent_material_idx']
+    const_ori_idx = volume_element['constituent_orientation_idx']
+    const_mat_frac = volume_element['constituent_material_fraction']
+    const_phase_lbl = volume_element['constituent_phase_label']
+    quats = volume_element['orientations']['quaternions']
```

Comment:
use of temporary variables for each array in volume_element - is there a better way?
This way the concatinate lines are shorter (instead of using `volume_element['']`) each time.

Reply:
This is also fine 👍